ChangeLog.rst

```diff
@@ -1,3 +1,88 @@
+2017-04-27 azure 2.0.0rc7
```

Comment:
The change in setup.py is changing the version from 2.0.0rc7 to 2.0.0.  So is this supposed to be 2.0.0 instead?